versioned_docs/version-8.1/components/connectors/out-of-the-box-connectors/aws-sns-inbound.md

```diff
@@ -0,0 +1,156 @@
+---
+id: aws-sns-inbound
+title: Amazon Simple Notification Inbound Service Connector
+sidebar_label: Amazon SNS Inbound Connector
+description: Subscribe to your AWS SNS topic notification
+---
+
+The **Amazon Simple Notification Service (SNS) Inbound Connector** is a Connector that allows you to start or continue
+a BPMN process triggered by an [AWS SNS](https://console.aws.amazon.com/sns/home) notification.
```

Comment:
does it make sense to mention on which protocol it is based? or is it irrelevant from a technical point of view?

Reply:
Added to descriptions